product-cli/cmd/cluster/agent/create.go

```diff
@@ -0,0 +1,48 @@
+package agent
+
+import (
+	"context"
+
+	"github.com/openshift/hypershift/cmd/cluster/agent"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/spf13/cobra"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "agent",
+		Short:        "Creates basic functional HostedCluster resources on Agent",
+		SilenceUsage: true,
+	}
+
+	opts.AgentPlatform = core.AgentPlatformCreateOptions{
+		APIServerAddress: "",
+		AgentNamespace:   "",
+	}
+
+	cmd.Flags().StringVar(&opts.AgentPlatform.APIServerAddress, "api-server-address", opts.AgentPlatform.APIServerAddress, "The API server address that should be used for components outside the control plane")
```

Comment:
Is there anything else we can add to make this more clear? what are components outside the control plane?

Reply:
@avishayt @eranco74 - is there anything additional here we can add?

Reply:
This is the hosted cluster kubeapi endpoint.   
Perhaps
```The API server address is the IP address for Kubernetes API communication.```